pkg/storage/secret/metadata/data/secure_value_read.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  {{ .Ident "guid" }},
+  {{ .Ident "name" }},
+  {{ .Ident "namespace" }},
+  {{ .Ident "annotations" }},
+  {{ .Ident "labels" }},
+  {{ .Ident "created" }},
+  {{ .Ident "created_by" }},
+  {{ .Ident "updated" }},
+  {{ .Ident "updated_by" }},
+  {{ .Ident "status_phase" }},
+  {{ .Ident "status_message" }},
+  {{ .Ident "title" }},
+  {{ .Ident "keeper" }},
+  {{ .Ident "decrypters" }},
+  {{ .Ident "ref" }},
+  {{ .Ident "external_id" }}
+FROM
+  {{ .Ident "secret_secure_value" }}
+WHERE 1 = 1 AND
+  {{ .Ident "namespace" }} = {{ .Arg .Namespace }} AND
+  {{ .Ident "name" }} = {{ .Arg .Name }}
+ORDER BY {{ .Ident "updated" }} DESC
```

Comment:
Does this needs an `ORDER BY`? I worry that because of the `ORDER BY`, an index might not be used? We expect it to return only `1` item anyways.

Reply:
you are right, I will remove it

Reply:
@leandro-deveikis if you can also remove it from the keeper one 🙈 (i see the same thing)